packages/react-components/react-avatar/src/components/AvatarGroup/AvatarGroup.test.tsx

```diff
@@ -2,24 +2,133 @@ import * as React from 'react';
 import { AvatarGroup } from './AvatarGroup';
 import { AvatarGroupItem } from '../AvatarGroupItem';
 import { isConformant } from '../../common/isConformant';
-import { render } from '@testing-library/react';
+import { render, screen, within } from '@testing-library/react';
 
 describe('AvatarGroup', () => {
-  // TODO: Remove component-has-static-classnames-object from disabled tests.
   isConformant({
     Component: AvatarGroup,
     displayName: 'AvatarGroup',
     disabledTests: [
       'component-has-static-classname',
       'component-has-static-classname-exported',
+      // AvatarGroup's overflowContent is rendered in a portal, therefore this test won't work
```

Comment:
You can test this but you have to override some functions for the conformance tests. I have an issue to make this better but I haven't had time to look into a general purpose solution.

In the mean time, [Tooltip is an example of how to conformance test Portals](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-tooltip/src/components/Tooltip/Tooltip.test.tsx#L8-L28)

Reply:
@spmonahan I think for this case, the test should be disabled. I was able to get the custom function working, but I think that's not what we should be doing. When adding the getAvatarGroupElement target function we are saying "test isConformant but first get this element and test against the result". The problem with this is that we are then focusing on the PopoverSurface. This is why I disabled it since I think it's more worthwhile that we test the root and ref rather than just test the overflow content which doesn't handle many of the other tests.

One thing I would propose would be to add a second isConformat test that will only test the overflowContent part of AvatarGroup and leave the isConformant above as is. Seems hacky, but it would cover all component. 

What are your thoughts on this?

On a side note, this brings then another case that we should consider in our conformance tests.

Reply:
I'm fine with disabling in this case. I'm not sure the best way to generally handle conformance for situations like this but I also haven't had time to give it much thought. It's looking like we'll have a few different components that make use of Portal by the time I get back to it so hopefully we'll be able to devise a generic solution.